src/algebra/big_operators/finsupp.lean

```diff
@@ -50,3 +51,10 @@ lemma finsupp.mul_sum (b : S) (s : α →₀ R) {f : α → R → S} :
 by simp only [finsupp.sum, finset.mul_sum]
 
 end
+
+namespace nat
+
```

Comment:
Can you add a comment to explain the statement in a familiar way for a mathematician? `f.prod pow` is pretty hard to understand.